game/templates/game/board.html

```
@@ -331,7 +350,7 @@
             </div>
         </div>
 
-    </div>
+    </>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)